src/lints/function_export_name_changed.ron

```diff
@@ -0,0 +1,55 @@
+SemverQuery(
+    id: "function_export_name_changed",
+    human_readable_name: "function's export name has changed",
+    description: "A function's ABI name with #[no_mangle] or #[export_name = \"name\"] has changed",
+    required_update: Major,  // Not specified in rust semver doc? most similar is for #[repr] changes: https://doc.rust-lang.org/cargo/reference/semver.html#type-layout
```

Comment:
Yeah this is definitely major, and using the `#[no_mangle]` doc link as a reference is a reasonable place for now until the cargo semver docs get a section for this.